packages/web/src/pages/upload-page/fields/MultiTrackSidebar.module.css

```diff
@@ -0,0 +1,99 @@
+.root {
+  /* Required for Safari */
+  position: -webkit-sticky;
+  position: sticky;
```

Comment:
nice, double check that -webkit-sticky doesn't get generated automatically by css-modules?